### ISSUE TITLE
Audio Dropouts on LineageOS 17

### ISSUE THREAD

**Issue by himijendrix24** [y20k/transistor#348]
**Describe the bug**
Since updating to LineageOS 17 I have many audio dropouts on all stations. 
This occurs also with perfect wifi connection.
Every 1-2 Minute there are very short breaks of some seconds, after which playback continues.
If I listen to the stations via a browser there is no problem, so I guess it might be a buffering problem.

In the past I had Lineage OS 16 and transistor worked good there.
I also tested it on the same device using the stock android 9 and there it works without problems.

**Device:**
 - Device: Moto X4
 - OS: Lineage OS 17.1
 - Version: 4.0.11

**Comment by comradekingu** [comment]
Can confirm.
    Device: Moto G7 Power
    OS: Lineage OS 17.1
    Version: 4.0.16

**Comment by Sanphobia** [comment]
I do not believe this is a lineage OS issue, nor is it dependent on brand. I have submitted #402 as I have neither issue nor #355

**Comment by Sanphobia** [comment]
*neither lineage nor a Motorola
